apps/web-client/src/routes/orders/customers/[id]/+page.svelte

```diff
@@ -8,30 +8,100 @@
 	import CustomerOrderMetaForm from "$lib/forms/CustomerOrderMetaForm.svelte";
 	import { customerOrderSchema } from "$lib/forms";
 
-	import { data } from "../data";
 	import { getOrderLineStatus } from "$lib/utils/order-status";
+	import type { PageData } from "./$types";
+	import { page } from "$app/stores";
+	import { addBooksToCustomer, upsertCustomer } from "$lib/db/orders/customers";
+	import { onDestroy, onMount } from "svelte";
+	import { invalidate } from "$app/navigation";
+	import { writable, get, type Writable } from "svelte/store";
+	import type { CustomerOrderLine } from "$lib/db/orders/types";
+	import type { BookEntry } from "@librocco/db";
+	import { createIntersectionObserver } from "$lib/actions";
 
-	import Page from "$lib/components/Page.svelte";
-	import { view } from "@librocco/shared";
+	export let data: PageData;
 
-	const { customers, customerOrderLines, books } = data;
+	const { customer, customerOrderLines, books, ordersDb } = data;
+
+	const id = parseInt($page.params.id);
+	let currentBookISBN = "";
+
+	// #region infinite-scroll
+	let maxResults = 20;
+	// Allow for pagination-like behaviour (rendering 20 by 20 results on see more clicks)
+	const seeMore = () => (maxResults += 20);
+	// We're using in intersection observer to create an infinite scroll effect
+	const scroll = createIntersectionObserver(seeMore);
+	// #endregion infinite-scroll
+
+	// #region reactivity
+	let disposer: () => void;
+	onMount(() => {
+		// Reload add customer data dependants when the data changes
+		const disposer1 = data.ordersDb.rx.onPoint("customer", BigInt(id), () => invalidate("customer:data"));
+		// Reload all customer order line/book data dependants when the data changes
+		const disposer2 = data.ordersDb.rx.onRange(["customer_order_lines", "customer_supplier_order"], () => invalidate("customer:books"));
+
+		disposer = () => (disposer1(), disposer2());
+	});
+	onDestroy(() => {
+		// Unsubscribe on unmount
+		disposer();
+	});
+
+	// I see the error of my ways: This is a terrible way to update a persisted value but is necessary for the time being bcs of the way TextEditable operates
+	// TODO: replace with form sumission or, at least, an imperative update
+	const createFieldStore = <T extends string | number>(init: T, onUpdate: (x: T) => Promise<any> | any): Writable<T> => {
```

Comment:
We are not using TextEditable for these properties any more, so this can be removed